packages/apps/graph/src/services/account-service.ts

```diff
@@ -33,3 +40,36 @@ export async function getChainFungibleAccount({
       }
     : null;
 }
+
+export async function getChainNonFungibleAccount({
+  chainId,
+  accountName,
+}: {
+  chainId: string;
+  accountName: string;
+}): Promise<ChainNonFungibleAccount | null> {
+  const accountDetails = await accountDetailsLoader.load({
```

Comment:
These can be run in parallel.

Also, this looks like the same code as here: https://github.com/kadena-community/kadena.js/pull/1399/files#diff-3e16f18f50f6e4532441c076c425dfa2a05525e408f9804d1355374c16d8debbR23

Is it possible to remove the duplication?